src/V3Os.h

```diff
@@ -58,7 +58,8 @@ class V3Os {
     static vluint64_t rand64(vluint64_t* statep);
     static string trueRandom(size_t size);
 
-    // METHODS (performance)
+    // METHODS (time & performance)
+    static void usleep(int64_t usec);  ///< Sleep for a given number of microseconds.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This C routine is considered obsolete (as opposed to the shell command by the same name).   The interaction of this function with SIGALRM and other timer functions such as sleep(), alarm(), setitimer(), and nanosleep() is unspecified (CWE-676).  Use nanosleep(2) or setitimer(2) instead. ](https://app.codacy.com/app/wsnyder/verilator/pullRequest?prid=4719042)